account_payment_order_return/tests/test_account_payment_order_return.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2017 Tecnativa - Luis M. Ontalba
+# Copyright 2021 Tecnativa - João Marques
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0
+
+from odoo import fields
+from odoo.tests import common
+from odoo.tests.common import Form
+
+
+class TestAccountPaymentOrderReturn(common.SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestAccountPaymentOrderReturn, cls).setUpClass()
+        cls.account_type = cls.env["account.account.type"].create(
+            {
+                "name": "Test Account Type",
+                "type": "receivable",
+                "internal_group": "income",
+            }
+        )
+        cls.a_receivable = cls.env["account.account"].create(
+            {
+                "code": "TAA",
+                "name": "Test Receivable Account",
+                "reconcile": True,
+                "internal_type": "receivable",
+                "user_type_id": cls.account_type.id,
+            }
+        )
+        cls.partner = cls.env["res.partner"].create(
+            {"name": "Test Partner 2", "parent_id": False}
```

Comment:
```suggestion
            {"name": "Test Partner 2"}
```
IMO parent_id isn't need to define False (because it's default value).

Reply:
Thanks @victoralmau 
Change made